customer/autostartup/shutdown.sh

```diff
@@ -7,10 +7,11 @@ kubectl config use-context $KUBE_CONTEXT
 
 echo "Stopping Replicated"
 REPLICATED_POD_ID=$(kubectl get pod -l "app=replicated,tier=master" -o name | sed 's/pod\///')
-kubectl exec $REPLICATED_POD_ID -c replicated -- replicatedctl app stop --force
+kubectl exec $REPLICATED_POD_ID -c replicated -- replicatedctl app stop --force --attach
 
+# Prevents issues on shutdown due to fs-snapshotter deployment
 echo 'Delete replicated-shared-fs-snapshotter deployment'
-kubectl delete deployments/replicated-shared-fs-snapshotter
+kubectl delete deployments/replicated-shared-fs-snapshotter || true 
```

Comment:
can we add a comment about why this is okay to ignore

Reply:
Done